.github/workflows/bump-golang.yml

```diff
@@ -13,12 +13,13 @@ jobs:
   bump:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/workflows/bump-golang
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/bump-golang
         with:
           branch: 'main'
           # NOTE: when a new golang version please update me with 1.<go-version>
           go-minor: '1.22'
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          command: '--experimental apply'
```

Comment:
This input doesn't seems to be present in the `./.github/actions/bump-golang` workflow.

Reply:
good catch!

https://github.com/elastic/golang-crossbuild/pull/403/commits/812759ab6be0eaa05620da9126aef2d4459b5c81